### DISCUSSION TITLE
How to access global loading state?

### DISCUSSION THREAD

**Discussion by Eraldo** [the-guild-org/apollo-angular#1555] [General]
How do I check if there are any queries currently in flight?

So far I have used: https://github.com/kamilkisiela/apollo-angular-link-network-status/issues/5
However this library is not compatible with Apollo v3.
(I have tried doing the upgrade myself locally -- but failed)

Any ideas of how I can check for that global loading status? 😄 
Assistance and ideas are greatly appreciated. 🙏 

Have a great day!

**Comment by kamilkisiela** [comment]
https://github.com/kamilkisiela/apollo-angular-link-network-status/pull/6

I will release a proper version tomorrow

**Reply by Eraldo** [reply]
Awesome! Thank you. 🙏

**Reply by Eraldo** [reply]
Still not working! :|

**Reply by Eraldo** [reply]
I'm still waiting. ;)

**Reply by Eraldo** [reply]
Is this still planned?

**Comment by Eraldo** [comment]
Still seems to be stuck. :|

**Reply by Eraldo** [reply]
Any update?
